glitter/publisher/utils.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.utils import timezone
+
+from .models import PublishAction
+
+
+def process_actions(action_ids=None):
+    """
+    Process actions in the publishing schedule.
+
+    Returns the number of actions processed.
+    """
+    actions_taken = 0
+    action_list = PublishAction.objects.prefetch_related(
```

Comment:
Is this helps anyway by doing prefetch_related on the `content_object`  I find out that SQL still remains the same if I use `prefetch_related` ?